tree.dot

```diff
@@ -190,6 +190,7 @@ digraph G {
 		"Goon 2009 Public SVN"          -> "Vagabond Station 13"
 		"Goon 2016 Release"             -> "Goon 2020 Release" [color = red]
 		"Goon 2016 Release"             -> "T/Goonstation"
+		"Goon 2020 Release"		-> "CoolStation"
```

Comment:
This is incorrect. Goon 2020 Release != 29th August 2021 - you want `Goonstation (Open Source)`

Reply:
Aha! Understood, changed.